### PR TITLE
Continuous deployment

### PR DESCRIPTION
**Pull request by sidpremkumar**
Merging the CD branch into master so we can keep them the same. This PR is all the changes I made while deploying. 

Left commits because it indicates better what I was working on then a single squashed commit but open to changing that.

### CODE REVIEW

**Comment by sidpremkumar** [comment]
Relates to JIRA: DEVOPSA-7103

File: continuous-deployment/install_rhmsg.sh [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
-    # Since we don't trust any internal CAs at this point, we must connect over
-    # http
-    sed -i 's/https:/http:/g' $repo_file
+main() {
+  if [ -n "$RCM_TOOLS_REPO" ]; then
+      repo_file=/usr/local/src/sync2jira/continuous-deployment/rcm-tools-fedora.repo
+      curl -L -o $repo_file $RCM_TOOLS_REPO
+      # Since we don't trust any internal CAs at this point, we must connect over
+      # http
+      sed -i 's/https:/http:/g' $repo_file
```
Comment (ralphbean) [inline continuous-deployment/install_rhmsg.sh]: What prevents us from trusting the internal CA before this point?

Comment (sidpremkumar) [reply continuous-deployment/install_rhmsg.sh]: We add the internal CA here: https://github.com/sidpremkumar/Sync2Jira/pull/83/files#diff-9d8e13c2dac78d2e51e7ad1eebff3f95R52

So prior to this, before the container starts, we don't have the CA_URL needed to get the certs. 

Used this as a guide: https://github.com/fedora-modularity/message-tagging-service/blob/master/docker/install-dependencies.sh

Comment (ralphbean) [reply continuous-deployment/install_rhmsg.sh]: Ack.  We trust the CA at runtime, but here we are working at build time.

**Pull request merged**